src/components/adjudications/AdjudicationForm.jsx

```diff
@@ -50,19 +50,14 @@ class AdjudicationForm extends React.Component {
     const { 
       adjudicationId, 
       collectionName, 
-      currentValues : {
-        audioURL, 
-        judgeName, 
-        notes
-      } 
+      currentValues
     } = this.props;
     const { artisticMark, technicalMark } = this.state;
     const cumulativeMark = (parseInt(artisticMark, 10) + parseInt(technicalMark, 10)) / 2;
+
     const data = {
-      audioURL,
       cumulativeMark,
-      judgeName,
-      notes,
+      ...currentValues,
```

Comment:
So the PR comment says this will restrict undefined values from being sent to firebase. Is it possible that currentValues will have something as `undefined`?

Reply:
I don't believe that `currentValues` will hold any undefined values.

`currentValues` is created in AdjudicationsSection.jsx and passed down through several children until it reaches AdjudicationForm.jsx. I traced the contents of currentValues as it is being passed and it seems that if a field is not defined in Firebase, it will not appear in `currentValues`.